about.html

```diff
@@ -12,62 +12,96 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
+  <STYLE>A {text-decoration: none;} </STYLE>
```

Comment:
Interesting way to remove the text decoration here, rather than just doing it in the CSS with the same property on the `a` tags alone. I wouldn't recommend doing this because it can break the rest of how things work since this is a style that will be applied globally on the page.